go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -97,6 +117,9 @@ func (c *addressManager) Run(stopChan <-chan struct{}) {
 						klog.Errorf("Failed to set node annotations: %v", err)
 					}
 				}
+			case <-addressSyncTimer.C:
+				klog.Info("Node IP manager calling sync() explicitly")
```

Comment:
Should we increase the log level here? If there is no activity on the node it would print it every 30s

Reply:
Agree. let me push a quick update for that here. Thanks @Kyrtapz !